pom.xml

```diff
@@ -112,12 +112,6 @@
             <version>1.2.3</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
Please be careful here. Our testcontainers module has an old version of this library too, and it has some major bugs. We need to change this dependency to 1.17.3 or higher, but should not remove it.

Reply:
In my opinion we need to have this dependency only in `cartridge-java-testcontainers`

Reply:
Done